suricata.yaml.in

```diff
@@ -896,17 +897,40 @@ app-layer:
       #                           Limit to how many layers of compression will be
       #                           decompressed. Defaults to 2.
-      #
-      # server-config:            List of server configurations to use if address matches
-      #   address:                List of IP addresses or networks for this block
-      #   personalitiy:           List of personalities used by this block
-      #   request-body-limit:     Limit reassembly of request body for inspection
-      #                           by http_client_body & pcre /P option.
-      #   response-body-limit:    Limit reassembly of response body for inspection
-      #                           by file_data, http_server_body & pcre /Q option.
-      #   double-decode-path:     Double decode path section of the URI
-      #   double-decode-query:    Double decode query section of the URI
-      #
+      #   path-convert-backslash-separators:
+      #                           Converts backslashes to slashes in path (URI)
+      #                           Default is true with IDS personality
+      #   path-convert-lowercase:
+      #                           Converts path (URI) to lowercase
+      #                           Default is true with IDS personality
+      #   path-nul-encoded-terminates:
+      #                           Ends a path (URI) on an encoded nul character
+      #                           Default is false
+      #   path-nul-raw-terminates:
+      #                           Ends a path (URI) on a raw nul character
+      #                           Default is false
+      #   path-separators-compress:
+      #                           Concatenates together consecutive separators in a path
+      #                           Default is true with IDS personality
+      #   path-separators-decode:
+      #                           Decodes encoded separators in path (ie %2F)
+      #                           Default is true with IDS personality
+      #   path-u-encoding-decode:
+      #                           Decodes unicode in path (ie %uHHHH)
+      #                           Default is true with IDS personality
+      #   path-url-encoding-invalid-handling:
+      #                           Behavior against an invalid encoding in path (such as %ZZ)
+      #                           Can be "preserve_percent","remove_percent", "decode_invalid"
+      #                           Default is "preserve_percent"
+      #   path-utf8-convert-bestfit:
+      #                           Decodes unicode, and replaces invalid characters with bestfit
+      #                           Default is true with IDS personality
+      #   path-bestfit-replacement-char:
+      #                           Selects character to replace invalid unicode
+      #                           Default is '?'
+      #   query-plusspace-decode:
+      #                           Decodes plus as space
+      #                           Default is true
```

Comment:
I think that these, and probably a bunch of the existing options, should not be in the default config. Instead we should have a comment like "for advanced options see XYZ", where XYZ a link to the user guide.

Reply:
ok